Makefile

```diff
@@ -62,4 +66,6 @@ checkfmt:
 
 .PHONY: update-mem-include
 update-mem-include:
-	cp -af ../tillitis-key1/hw/application_fpga/fw/tk1_mem.h include/tk1_mem.h
+	cp -af ../tillitis-key1/hw/application_fpga/fw/tk1_mem.h \
+	include/tkey/tk1_mem.h
+	echo "Remember to update header include gard!"
```

Comment:
Do you mean "guard"?

Reply:
Yes. Fixed.

Reply:
Is it? Did you push your change? I'm still seeing "gard".